### DISCUSSION TITLE
What importing parameters does the official nominatim instance uses?

### DISCUSSION THREAD

**Discussion by guillaume** [mediagis/nominatim-docker#527] [Q&A] [answered]
If one were to replicate the official instance at https://nominatim.openstreetmap.org/ui/search.html

What parameters should be used when importing?
https://github.com/mediagis/nominatim-docker/blob/master/4.3/example.md

I can guess that `IMPORT_US_POSTCODES` and `IMPORT_GB_POSTCODES` are probably enabled.

But for instance, is `IMPORT_TIGER_ADDRESSES` enabled? And is IMPORT_WIKIPEDIA favored?

There's already enough challenging trial and error to get a planet-wide import up and running so it would be useful to get that information.

They may not use the docker containers for their setup/infra, in that case i'm interested in the conceptual equivalent. 

Thanks

**Comment by mtmail** [comment, marked as answer]
nominatim.openstreetmap.org runs the git master version, not stable releases.

> I can guess that IMPORT_US_POSTCODES and IMPORT_GB_POSTCODES are probably enabled.

yes and yes

> But for instance, is IMPORT_TIGER_ADDRESSES enabled? And is IMPORT_WIKIPEDIA favored?

yes and yes

> They may not use the docker containers for their setup

They don't.

Import style used is `extra`, replication is minutely.
